TravelerApp/TravelerAppCore/Controller/JSON.cs

```diff
@@ -38,13 +39,36 @@ public static void Read(List<Root> targetData)
                 }
             }
             stopper.Stop();
+            Console.WriteLine("\n----------------------------------------------------");
             Console.WriteLine($"Czas odczytu wszystkich plików: {stopper.Elapsed}");
             Console.WriteLine($"Ilość odczytanych plików: {targetData.Count()}");
             Console.WriteLine("Koniec deserializacji");
+            Console.WriteLine("----------------------------------------------------\n");
         }
-        public static void Write(List<Root> targetData)
+
+        public static void Write()
```

Comment:
ta metoda write robi znacznie więcej rzeczy niż zapisywanie do pliku, np zbiera również informacje o nowym hotelu. Tutaj tak naprawdę oczekiwał bym że metoda Write w parametrze przyjmie obiekt Root i zapisze go do pliku. W metodzie write nie powinno mnie interesować skąd ten parametr przyjdzie i gdzie powstał. Dzięki temu jak np będę miał aplikacje webową to dalej będę mógł użyć metody write.

Reply:
Faktycznie, zgadzam się. Metoda Write mogłaby też przyjmować tylko string zamiast obiektu i wówczas byłaby bardziej uniwersalna. W mainie mógłbym natomiast do tej zmiennej zrobić serializację obiektu z wykorzystaniem` JsonConvert.SerializeObject(new Hotel().CreateNew());` i takiego stringa uczynić parametrem tej metody.